clients/drcachesim/tracer/raw2trace.cpp

```diff
@@ -1002,6 +1019,13 @@ raw2trace_t::process_header(raw2trace_thread_data_t *tdata)
     create_essential_header_entries(buf, version, tdata->file_type, tid, pid);
     buf += trace_metadata_writer_t::write_marker(buf, TRACE_MARKER_TYPE_CACHE_LINE_SIZE,
                                                  header.cache_line_size);
+    if (header.vector_length_bytes > 0) {
+#ifdef AARCH64
+        dr_set_sve_vector_length(header.vector_length_bytes * 8);
```

Comment:
We'll have to udpate internal tools as well; along with other users who have their own tools.  Please add this to the release notes so it's visible.

Probably it should be added to the docs too: to sec_drcachesim_format where it talks about the encoding_is_new field, etc.?

Reply:
Done.

Reply:
Should add this note to the marker docs too.

Reply:
Done.